py/torch_tensorrt/dynamo/_compiler.py

```diff
@@ -221,7 +225,7 @@ def compile(
     logger.debug("Input graph: " + str(gm.graph))
 
     # Apply lowering on the graph module
-    gm = post_lowering(gm)
+    gm = post_lowering(gm, use_fp32_acc=use_fp32_acc)
```

Comment:
Should a check be added to check whether only fp16 is the only precision in `enabled_precisions` and raise an error if it's not?

Reply:
Yes. I'm planning to add that. This is currently WIP.